.github/workflows/publish.yml

```diff
@@ -0,0 +1,21 @@
+name: Publish
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: Brightspace/third-party-actions@actions/checkout
+      - name: Setup Node
+        uses: Brightspace/third-party-actions@actions/setup-node
+      - name: Install dependencies
+        run: npm ci --ignore-scripts
+      - name: Build
+        run: npm run build
+      - name: Publish
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: npm publish --ignore-scripts
```

Comment:
This package wasn't previously using semantic-release, but should it? Not really up-to-date on the details there. (This also rarely gets updated, which might change the answer to that question.)

Reply:
I think it should yeah. Otherwise there's currently nothing in here that will increment the version in `package.json`... that must have been a manual thing previously. I can help get that set up tomorrow and it's going to require a special `D2L_GITHUB_TOKEN` to bypass the new branch protection rules that the security team will be enforcing shortly.

Reply:
Aha, yes it was - hence the note I added to the readme here, actually. Being able to just use the GH Releases tool rather than having to manually run `npm version` is much nicer, so I'm definitely good with that! Thanks :+1: